lib/SimplePie/SimplePie/Item.php

```diff
@@ -2896,7 +2862,6 @@ public function get_enclosures()
 					$width = null;
 
 					$url = $this->sanitize($enclosure[0]['attribs']['']['url'], SIMPLEPIE_CONSTRUCT_IRI, $this->get_base($enclosure[0]));
-					$url = $this->feed->sanitize->https_url($url);	//FreshRSS
```

Comment:
Line deleted by mistake. Fixed in https://github.com/FreshRSS/FreshRSS/pull/3568